dbt/include/athena/macros/utils/datatypes.sql

```diff
@@ -12,3 +12,7 @@
 {%- macro athena__type_int() -%}
     INTEGER
 {%- endmacro -%}
+
+{%- macro athena__type_string() -%}
+    VARCHAR
+{%- endmacro -%}
```

Comment:
New line please!

Reply:
Done! Sorry

Reply:
Ups, now you have one extra. The ci is still failing.